api/envoy/api/v2/eds.proto

```diff
@@ -43,9 +43,12 @@ service EndpointDiscoveryService {
 //
 // With EDS, each cluster is treated independently from a LB perspective, with
 // LB taking place between the Localities within a cluster and at a finer
-// granularity between the hosts within a locality. For a given cluster, the
-// effective weight of a host is its load_balancing_weight multiplied by the
-// load_balancing_weight of its Locality.
+// granularity between the hosts within a locality. The load balancing weight
```

Comment:
Maybe it makes more sense to explain locality selection first? I think it might help people understand the host selection model better

Reply:
Yeah I think that makes much more sense. Let me know how it looks now